redi-connect-front/src/App.tsx

```diff
@@ -1,54 +1,52 @@
-import React, { useEffect, Suspense } from 'react';
-import { createMuiTheme, MuiThemeProvider } from '@material-ui/core/styles';
-
-import { Provider as StoreProvider } from 'react-redux';
-import './App.scss';
-import { history, Router } from './services/history/history';
-import { Routes } from './components/Routes';
-import { store } from './redux/store';
-import { profileFetchStart } from './redux/user/actions';
-import AppNotification from './components/AppNotification';
+import React, { useEffect, Suspense } from "react";
+import { createMuiTheme, MuiThemeProvider } from "@material-ui/core/styles";
+import { Provider as StoreProvider } from "react-redux";
+import { history, Router } from "./services/history/history";
+import { Routes } from "./components/Routes";
+import { store } from "./redux/store";
+import { profileFetchStart } from "./redux/user/actions";
+import AppNotification from "./components/AppNotification";
 
 const theme = createMuiTheme({
   palette: {
     primary: {
-      main: '#58adc4',
-      contrastText: '#fff',
+      main: "#58adc4",
```

Comment:
Recommend that we avoid code style changes until we put in place one single set of linting rules and run them on the entire codebase. That should ideally be done when everybody's feature branches are merged into `develop` so we avoid merge conflict headaches.

@ezonai how about you coordinate this?

We should also make sure the linting rules are automagically picked up by everyone's IDE so we don't get undesirable "auto-format-code-on-save" behaviour.